### DISCUSSION TITLE
Raspberry Pi Pico

### DISCUSSION THREAD

**Discussion by jmsunseri** [darthcloud/BlueRetro#685] [Q&A]
What would it take to port this over so that it would work on a Raspberry Pi Pico W?

**Comment by Invictaz** [comment]
Probably a lot of effort since it now is ESP32 based and Blueretro uses a lot of RAM that the Pico does not have 

![image](https://github.com/darthcloud/BlueRetro/assets/20338545/5cc5eff1-2e66-48b9-aae3-5f53f7b46403)
